### ISSUE TITLE
Some more problem(s)

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/galaxy-language-server#223]
I often observe that closing tags are not added automatically.

Also I see the follwoing (no idea if this is connected)

```
  Message: TypeError: TextDocumentPositionParams.__init__() got an unexpected keyword argument 'textDocument'
  Code: -32602 
{'traceback': ['  File "/home/berntm/.vscode-oss/extensions/davelopez.galaxy-tools-0.10.1-universal/glsenv/lib/python3.10/site-packages/pygls/protocol.py", line 338, in _handle_request\n    handler(params, msg_id)\n', '  File "/home/berntm/.vscode-oss/extensions/davelopez.galaxy-tools-0.10.1-universal/glsenv/lib/python3.10/site-packages/pygls/protocol.py", line 83, in decorator\n    ret_val = base_func(self, *args, **kwargs)\n', '  File "/home/berntm/.vscode-oss/extensions/davelopez.galaxy-tools-0.10.1-universal/glsenv/lib/python3.10/site-packages/pygls/protocol.py", line 801, in lsp_workspace__execute_command\n    self._execute_request(msg_id, cmd_handler, params.arguments)\n', '  File "/home/berntm/.vscode-oss/extensions/davelopez.galaxy-tools-0.10.1-universal/glsenv/lib/python3.10/site-packages/pygls/protocol.py", line 264, in _execute_request\n    self._send_response(msg_id, handler(params))\n', '  File "/home/berntm/.vscode-oss/extensions/davelopez.galaxy-tools-0.10.1-universal/glsenv/lib/python3.10/site-packages/galaxyls/server.py", line 193, in auto_close_tag\n    params = deserialize_command_param(parameters[0], TextDocumentPositionParams)\n', '  File "/home/berntm/.vscode-oss/extensions/davelopez.galaxy-tools-0.10.1-universal/glsenv/lib/python3.10/site-packages/galaxyls/utils.py", line 39, in deserialize_command_param\n    obj = type(**params_dict)  # type: ignore [call-arg]\n']}
```

**Comment by davelopez** [comment]
Thanks! I'll investigate :+1:

**Comment by davelopez** [comment]
Thank you once more for reporting this! It should be fixed in the freshly released v0.10.2 version :+1:
